components/FormComponents.js

```diff
@@ -0,0 +1,226 @@
+        readOnly
+      />
+    </Group>
+  );
+}
+
+const Selection = styled.div`
+  position: relative;
+  z-index: 1000;
+`;
+const SelectionHeader = styled.div`
+  background: var(--primary-gradient);
+  color: var(--white);
+  padding: ${({ inputIcon, iconBefore }) =>
+    inputIcon && iconBefore ? "14px 12px 14px 46px" : "14px 46px 14px 12px"};
+  position: relative;
+  cursor: pointer;
+  border-radius: ${({ isOpen }) => (isOpen ? "13px 13px 0 0" : "13px")};
+`;
+const SelectionList = styled.ul`
+  list-style-type: none;
+  position: absolute;
+  top: 100%;
+  left: 0;
+  width: 100%;
+`;
+const SelectionOption = styled.li`
+  background: var(--card-bg);
+  color: var(--white);
+  padding: ${({ inputIcon, iconBefore }) =>
+    inputIcon && iconBefore ? "14px 12px 14px 46px" : "14px 46px 14px 12px"};
+  cursor: pointer;
+
+  &:last-child {
+    border-radius: 0 0 13px 13px;
+  }
+`;
+
+// function Select({
+//   children,
+//   name,
+//   labelText,
+//   inputIcon,
+//   iconBefore = true,
+//   options,
+//   handleChange,
+// }) {
+//   const [currentlySelected, setCurrentlySelected] = useState(options[0]);
+//   const [selectionOpen, setSelectionOpen] = useState(false);
+
+//   function openSelection() {
+//     setSelectionOpen((oldStatus) => !oldStatus);
+//   }
+//   function handleOptionChange(event, optionId) {
+//     if (optionId !== currentlySelected.id) {
+//       event.stopPropagation();
+//       const newCurrentlySelected = options.filter(
+//         (option) => option.id === optionId
+//       )[0];
+
+//       setCurrentlySelected((oldCurrentlySelected) => newCurrentlySelected);
+//     }
+//   }
+//   return (
+//     <Group>
+//       <Label>{labelText}</Label>
+//       <StyledSelection className="noselect" onClick={() => openSelection()}>
+//         {options.map((option, index) => (
+//           <StyledOption
+//             key={option.id}
+//             selectionOpen={selectionOpen}
+//             inputIcon={inputIcon}
+//             iconBefore={iconBefore}
+//             selected={option.id === currentlySelected.id}
+//             onClick={(event) => handleOptionChange(event, option.id)}
+//           >
+//             {index == 0 && (
+//               <InputIcon iconBefore={iconBefore}>
+//                 {inputIcon === "list" && <IconList />}
+//                 {inputIcon === "chevronDown" &&
+//                   (!selectionOpen ? <IconChevronDown /> : <IconChevronUp />)}
+//               </InputIcon>
+//             )}
+//             {option.name}
+//           </StyledOption>
+//         ))}
+//       </StyledSelection>
+//       <select
+//         name={name}
+//         defaultValue={children}
+//         onChange={(event) => handleChange(event)}
+//       >
+//         {options.map((option) => (
+//           <option key={option.id} value={option.id}>
+//             {option.name}
+//           </option>
+//         ))}
+//       </select>
+//     </Group>
+//   );
+// }
```

Comment:
Die ganzen Kommentare sind soweit ich weiß von den Coaches immer unerwünscht. pack die alle in einen .comments Folder und pack das in .gitignore z.B

Reply:
yes, vergessen zu entfernen. Danke